src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-if(CLR_CMAKE_TARGET_APPLE OR CLR_CMAKE_TARGET_FREEBSD OR CLR_CMAKE_TARGET_HAIKU)
+if(CLR_CMAKE_TARGET_APPLE OR CLR_CMAKE_TARGET_FREEBSD OR CLR_CMAKE_TARGET_HAIKU OR CLR_CMAKE_TARGET_BROWSER)
```

Comment:
I don't think we should set the CLR_CMAKE_USE_SYSTEM_LIBUNWIND to 1, but instead prevent compiling the libunwind explicitly.